style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+}
+
+header {
+  min-height: 100vh;
+  width: 100%;
+  background-image: linear-gradient(rgba(1, 9, 30, 0.7), rgba(4, 9, 30, 0.7)),
+    url(./images/banner.png);
+  background-position: center;
+  background-size: cover;
+  position: relative;
+}
+
+.menu {
+  display: flex;
+  padding: 2% 6%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 150px;
+}
+.logo a {
+  text-decoration: none;
+}
+.menu-links {
+  flex: 1;
+  text-align: right;
+}
+
+.menu-links ul li {
+  list-style: none;
+  display: inline-block;
+  padding: 8px 12px;
+  position: relative;
+}
+
+.menu-links ul li a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.menu-links ul li::after {
+  content: "";
+  width: 0%;
+  height: 2px;
+  background: #551a8b;
+  display: block;
+  margin: auto;
+  /* transition: 0.5s; */
+}
+
+.menu-links ul li:hover::after {
+  width: 100%;
+  transition: 0.5s;
+}
+
+.hero {
+  width: 90%;
+  color: #fff;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+.hero h1 {
+  font-size: 62px;
+  text-transform: capitalize;
+}
+.special {
+  color: #551a8b;
+  background: #fff;
+}
+.hero p {
+  margin: 10px 0 40px;
+  font-size: 14px;
+  color: #fff;
+}
+
+.hero-btn {
+  display: inline-block;
+  text-decoration: none;
+  color: #fff;
+  border: 1px solid #fff;
+  padding: 12px 34px;
+  font-size: 13px;
+  background: transparent;
+  position: relative;
+  cursor: pointer;
+}
+
+.hero-btn:hover {
```

Comment:
![good job](https://user-images.githubusercontent.com/123179856/228032161-3c32b1b8-3525-43bc-8b89-41d671a62672.gif)

Fantastic Job here!

Reply:
- [ ] I think its better if you explain why or what you believe is implemented well e.g "Fantastic job in creating a hover effect even though it was not required"